src/test/java/com/google/api/codegen/testdata/nodejs_doc_main_library.baseline

```diff
@@ -1159,8 +1159,11 @@ LibraryServiceApi.prototype.streamBooks = function(request, options) {
  *     // doThingsWith(response);
  * });
  * var name = '';
+ * var request = {
+ *     name : name
+ * };
  * // Write request objects.
- * stream.write({name: name});
+ * stream.write({request: request});
```

Comment:
This does not look correct. Please update nodejs method_sample.snip or somewhere else to keep the previous result.